src/components/RegForm/index.jsx

```diff
@@ -38,6 +38,18 @@ RegForm.SubTitle = function RegFormSubTitle ({ children, ...restProps }) {
 	)
 }
 
+RegForm.Frame = function RegFormFrame ({ children, ...restProps }) {
+	return (
```

Comment:
what is your thoughts on shrinking all styled components setup:
```js
// before
return (
    <Label { ...restProps }> { children } </Label>
)

// after
return (<Label { ...restProps }> { children } </Label>)
```

Reply:
It works, I'm just used to doing it that way. I think we can just do a one-liner too without the ( )